src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java

```diff
@@ -125,10 +125,21 @@ void shouldSuccessfullyFindFinancialTransaction_WhenFindingWithProperTransaction
         Assertions.assertEquals(financialTransactionDTO, foundTransaction);
     }
 
+    @Test
+    @DisplayName("when deleting financial transaction that does not exist should throw an exception")
+    void ShouldThrowAnException_WhenGivenTransactionDoesNotExist() {
+        //given
```

Comment:
I'm not sure there should be an empty line between //given and //when sections. Can you verify it?